instrumentation/apache-httpasyncclient-4.0/src/main/java/io/opentelemetry/instrumentation/auto/apachehttpasyncclient/ApacheHttpAsyncClientTracer.java

```diff
@@ -46,6 +47,11 @@ protected String method(HttpRequest request) {
     }
   }
 
+  @Override
+  protected @Nullable String flavor(HttpRequest httpRequest) {
```

Comment:
I like this placement (type annotation) 👍

Reply:
I don't :) It is IDEA's doing, so I have to accept :(

Reply:
Reformat code puts the annotation there? I haven't seen that before. I agree with @iNikem that for normal methods, annotation above looks a bit better mostly due to how annotations end up getting colored in IDE GitHub etc. @iNikem Is it not worth manually fixing these?

Reply:
Not reformat but "implement missing method", I think.

I don't want to fix this manually every time IDEA generates code for me. I can do it here, but it will not be sustainable

Reply:
ya, i like type annotations, but they are not common. i'm very surprised Intellij does the "right thing" 😉